src/scss/typography.scss

```diff
@@ -0,0 +1,57 @@
+////////////typography////////////////
+$spacer: 40px;
```

Comment:
you set value but don't use it

Reply:
Its need for bootstrap grid system

Reply:
Define all variables in separate file, like `variables.scss` and put only variables there. It will help to keep them organized